app/main/index.js

```diff
@@ -196,6 +196,28 @@ app.on('ready', () => {
 		app.quit();
 	});
 
+	// Show pdf in a new BrowserWindow
+	ipcMain.on('pdf-view', (event, url) => {
+		// Paddings for pdfWindow so that it fits into the main browserWindow
+		const paddingWidth = 55;
+		const paddingHeight = 22;
+
+		// Get the config of main browserWindow
+		const mainWindowState = global.mainWindowState;
+
+		// Window to view the pdf file
+		const pdfWindow = new electron.BrowserWindow({
+			x: mainWindowState.x + paddingWidth,
+			y: mainWindowState.y + paddingHeight,
+			width: mainWindowState.width - paddingWidth,
+			height: mainWindowState.height - paddingHeight,
+			webPreferences: {
+				plugins: true
```

Comment:
I think we need to pass `partition: 'persist:webviewsession'` here for realm session to pass to this BrowserWindow.